source/src/in/joind/DateHelper.java

```diff
@@ -16,13 +17,14 @@
      */
     public static String parseAndFormat(String input, String format) {
         // Assume that the input format is ISO8601
-        SimpleDateFormat dfOutput = new SimpleDateFormat(format), dfInput = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
+        SimpleDateFormat dfOutput = new SimpleDateFormat(format, Locale.US),
```

Comment:
OK I see - so would you not want to use the user's current locale instead? Or is this applicable only against the incoming date format?